manifests/b/Beyond-All-Reason/Beyond-All-Reason/1.2988.0/Beyond-All-Reason.Beyond-All-Reason.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Beyond-All-Reason.Beyond-All-Reason
+PackageVersion: 1.2988.0
+PackageLocale: en-US
+Publisher: Beyond All Reason

```

```suggestion
Publisher: BAR Team
```
This should match control panel

---

Should it only match here or should I change all the instances of this publisher?

---

I think it only has to match here, not the Publisher portion of the `PackageIdentifier`.